### ISSUE TITLE
Fira Mono italics

### ISSUE THREAD

**Issue by enlait** [mozilla/Fira#38]
Please make proper italics for Fira Mono! 
It is already one of the best fonts for programming (IMO), but seriously lacking without italics.

**Comment by krono** [comment]
(Is there a plan for this? It is the only thing that hinders me from using it as my coding font)

**Comment by cseelus** [comment]
👍🏾
@krono Same here.

**Comment by vikky49** [comment]
+1 for italics

**Comment by pank** [comment]
While we wait for an official italic variant of Fira Mono, Zachary Waldowski's mod can be used:

https://github.com/zwaldowski/Fira/tree/zwaldowski/mod-new/otf

**Comment by quicknir** [comment]
+1 for italics

**Comment by anthonymasure** [comment]
Please add italics!

**Comment by charlesroper** [comment]
+1

**Comment by slugovykh** [comment]
**+1** for _italics_

**Comment by ctongfei** [comment]
+1

**Comment by Mobyman** [comment]
+1

**Comment by mrobbetts** [comment]
+1

**Comment by fgilio** [comment]
Please +1

**Comment by reywright** [comment]
+1

**Comment by DivineDominion** [comment]
Now see https://github.com/carrois/Fira/issues/11

**Comment by Avi-D-coder** [comment]
I corrected a number of issues with @zwaldowski's fork see [fira-mono-italic](https://github.com/Avi-D-coder/fira-mono-italic)

**Comment by ackvf** [comment]
How is it going?

**Comment by Avi-D-coder** [comment]
It's not going. I recently stopped using fira code, my changes were made to the otf file directly, fira-code dropped otf.

**Comment by dsedivec** [comment]
> It's not going. I recently stopped using fira code, my chances were made to the otf file directly, fira-code dropped otf.

Can you say what you're using instead of Fira Code?

**Comment by Avi-D-coder** [comment]
Victor mono, jet brains mono and pragmata pro are all good options.

**Comment by somedeveloper00** [comment]
Any news yet?

**Comment by andrecasal** [comment]
No plans to add italics to Fira? 🤔

**Comment by tmillr** [comment]
Anyone home? 🤣

Darn.

**Comment by somedeveloper00** [comment]
bump

**Comment by kutsan** [comment]
If your contribution won't add any value to the discussion, please refrain from making comments. You're unnecessarily sending emails/notifications to people following the issue. Please use reactions, there is a reason they exist.

**Comment by somedeveloper00** [comment]
> If your contribution won't add any value to the discussion, please refrain from making comments. You're unnecessarily sending emails/notifications to people following the issue. Please use reactions, there is a reason they exist.

Actually the email may add contribution itself, notifying that this issue still exists and there are many who are looking forward to this. Sorry if the email/notification was of concern for you, know that it was not the intention at all.

**Comment by ackvf** [comment]
> > If your contribution won't add any value to the discussion, please refrain from making comments. You're unnecessarily sending emails/notifications to people following the issue. Please use reactions, there is a reason they exist.
> 
> Actually the email may add contribution itself, notifying that this issue still exists and there are many who are looking forward to this. Sorry if the email/notification was of concern for you, know that it was not the intention at all.

I don't think that posting a "bump" comment after another bump-like comment is necessary though, even if your argument is valid in the sense that the owners receive such notification too. And if you just want to subscribe to receive the email notifications yourself, you can always use the button on top right
![image](https://github.com/mozilla/Fira/assets/2313018/eda9320d-583f-4ea1-bc35-98182331bc42)



BTW, I have transitioned to this fork of FiraCode long ago. It comes with LOTS of customizations.
https://github.com/tonsky/FiraCode

**Comment by tmillr** [comment]
> > Actually the email may add contribution itself, notifying that this issue still exists and there are many who are looking forward to this. Sorry if the email/notification was of concern for you, know that it was not the intention at all.
> 
> I don't think that posting a "bump" comment after another bump-like comment is necessary though, even if your argument is valid in the sense that the owners receive such notification too. And if you just want to subscribe to receive the email notifications yourself, you can always use the button on top right
> 
> BTW, I have transitioned to this fork of FiraCode long ago. It comes with LOTS of customizations. https://github.com/tonsky/FiraCode

Ironically, your own comment appears to prove itself wrong, so I'm glad that it was bumped. (that fork doesn't appear to provide firacode or firamono italics though)

This issue is almost a decade old and doesn't appear to have (ever) been officially addressed? There were only a couple of comments/responses within the last few years, comments that also went unaddressed. This font is one of the most popular dev fonts, and this is a relatively popular issue.

If you don't want to receive notifications from those who are interested in this issue or its progress (or lack thereof), or from those who are nudging/asking for updates or more information etc., you are free to unsubscribe or manage your own notifications. The repo owner/org is also free to close, label, or address the issue.

**Comment by LiberalArtist** [comment]
> This issue is almost a decade old and doesn't appear to have (ever) been officially addressed? There were only a couple of comments/responses within the last few years, comments that also went unaddressed. This font is one of the most popular dev fonts, and this is a relatively popular issue.

In https://github.com/bBoxType/FiraSans/issues/4#issuecomment-699882058, the foundry wrote:

> Sadly, all support for the Fira project was withdrawn from Mozilla and Here some years ago.
> We then decided to host the project on our own to be able to keep it alive.
>
> But since we have no budget, it apears that we are not able to maintain such a big project as Fira as we would like to do.

My understanding is that Fira is not actively developed, though of course there are many forks and derived font families. Personally (I'm just another user!), I expect italics will only appear if someone opens a PR to contribute them or funds a designer to do so.
